### PR TITLE
msiRenameLocalZoneCollection and msiRenameCollection (4-2-stable)

### PR DESCRIPTION
**Pull request by korydraughn**

### CODE REVIEW

File: scripts/irods/test/test_iadmin.py [hunk 1]
```diff
@@ -1629,6 +1629,77 @@ def check_and_remove_rebalance_visibility_metadata(attempts):
         with session.make_session_for_existing_admin() as admin_session:
             admin_session.assert_icommand(['ils', '-l', '-r', self.test_dir], 'STDOUT_SINGLELINE', self.test_dir)
 
+class Test_Iadmin_modzone(session.make_sessions_mixin([('otherrods', 'rods')], [('alice', 'apass')]), unittest.TestCase):
```
Comment (korydraughn) [inline scripts/irods/test/test_iadmin.py:1632]: These tests are actually testing `msiRenameLocalZoneCollection` and should probably be moved to test_all_rules.py.

File: doxygen/microservices.cpp [hunk 2]
```diff
@@ -103,6 +103,7 @@
   - #msiDeleteCollByAdmin  - Deletes a collection by administrator
   - #msiRenameLocalZone  - Renames the local zone by updating various tables
```
Comment (trel) [inline doxygen/microservices.cpp:104]: does this need to be updated/removed?

Comment (korydraughn) [reply doxygen/microservices.cpp:104]: See `acRenameLocalZone` in the core.re.template.

Administrators will need to manually update their core.re files for this to work.

Comment (trel) [reply doxygen/microservices.cpp:104]: Okay - but new installs (where nearly all zone renaming happens...) would have the new core.re...

Do we need to update the implementation in the default_cpp rule engine too?

Comment (korydraughn) [reply doxygen/microservices.cpp:104]: I believe that change is in the PR

Comment (trel) [reply doxygen/microservices.cpp:104]: "I see" said the blind man.

**Comment by trel** [comment]
Convinced - please # when ready.

**Comment by korydraughn** [comment]
added #

**Pull request merged**